internal/controllers/synthesis/integration_test.go

```diff
@@ -321,3 +321,89 @@ func TestControllerSwitchingSynthesizers(t *testing.T) {
 		assert.NotEqual(t, comp.Status.CurrentSynthesis.ResourceSlices, initialSlices)
 	})
 }
+
+func TestControllerBackoff(t *testing.T) {
```

Comment:
Should we split these off as well? Maybe pod_backoff_test?

Reply:
I like it - done